prisma-fmt/tests/code_actions/test_api.rs

```diff
@@ -2,40 +2,58 @@ use lsp_types::{Diagnostic, DiagnosticSeverity};
 use once_cell::sync::Lazy;
 use prisma_fmt::offset_to_position;
 use psl::SourceFile;
-use std::{fmt::Write as _, io::Write as _, sync::Arc};
+use std::{fmt::Write as _, io::Write as _, path::PathBuf};
+
+use crate::helpers::load_schema_files;
 
 const SCENARIOS_PATH: &str = concat!(env!("CARGO_MANIFEST_DIR"), "/tests/code_actions/scenarios");
+/**
+ * File used as a target for code-actions in multi-file
+ * tests
+ */
+const TARGET_SCHEMA_FILE: &str = "_target.prisma";
```

Comment:
I'm not sure I fully understand the purpose of this as opposed to just having `a.prisma` & `b.prisma` or whatever else and using the loader

Reply:
Code actions are requested only for single file. So, when emulating lsp request during the tests
we need a way to designate that file somehow and put it's name into `CodeActionsParam` struct.